geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/MessageDispatcher.java

```diff
@@ -83,7 +83,7 @@ public class MessageDispatcher extends LoggingThread {
   /**
    * Default value in milliseconds for waiting for re-authentication
    */
-  private static final long DEFAULT_RE_AUTHENTICATE_WAIT_TIME = 5000;
+  private static final long DEFAULT_RE_AUTHENTICATE_WAIT_TIME = 60 * 1000;
```

Comment:
Please use `Duration` or `TimeUnit` to derive this value so that its very clear to the reader this is intended as 1 minute in milliseconds.

Reply:
```java
import static java.util.concurrent.TimeUnit.MINUTES;
...
private static final long DEFAULT_RE_AUTHENTICATE_WAIT_TIME = MINUTES.toMillis(1);
```